cloudsmith_cli/core/api/metrics.py

```diff
@@ -23,10 +46,13 @@ def entitlement_usage_metrics(owner=None, repo=None, **kwargs):
         {param: value for param, value in kwargs.items() if value is not None}
     )
 
+    headers = []
+    res = None
+
     if owner and repo:
         if hasattr(client, "metrics_entitlements_usage_list_with_http_info"):
             with catch_raise_api_exception():
-                res, _, headers = client.metrics_entitlements_usage_list_with_http_info(
+                res, _, headers = client.metrics_entitlements_usage_list0_with_http_info(
```

Comment:
Is this a typo, with the 0?

Reply:
nope, auto-generated bindings add a prefix of `0` which is super annoying - we reuse the base serializer for org metrics and repo metrics which is why we have this change here for the existing endpoint.